Mathlib/CategoryTheory/DifferentialObject.lean

```diff
@@ -28,10 +28,10 @@ universe v u
 
 namespace CategoryTheory
 
-variable (C : Type u) [Category.{v} C]
+variable (S : Type) [AddMonoidWithOne S] (C : Type u) [Category.{v} C]
```

Comment:
Did you intentionally restrict the universe of `S` here and elsewhere in this file? The file still seems to compile after generalizing.

Reply:
I'd hope that we don't ever have to use this for something other that Nat or Int, but you're right, we may as well. :-)